lighthouse-core/gather/driver.js

```diff
@@ -849,6 +849,21 @@ class Driver {
     });
   }
 
+  async listenForSecurityStateChanges() {
+    this.on('Security.securityStateChanged', state => {
+      this.lastSecurityState = state;
+    });
+    await this.sendCommand('Security.enable');
+  }
+
+  /**
+   * @return {LH.Crdp.Security.SecurityStateChangedEvent}
+   */
+  getSecurityState() {
+    // @ts-ignore
+    return this.lastSecurityState;
```

Comment:
should set this in the `Driver` constructor (I guess to null since `SecurityStateChangedEvent` is a relatively large object so it's probably not worth constructing an `unknown` starting value). Won't need to ts-ignore then

Reply:
It's ts-ignored to coerce a non-nullable value, since I believe this should always be set. I could remove it and allow this getter to be nullable, but would also need to handle that in the calling code (a case that I think will never happen). wdyt?

Reply:
If we believe it's always set but don't know for sure, WDYT about throwing an explicit error if it's not? 

This way it'll be clear to us when something unexpected happens and ts is happy by default :)